src/components/organisms/OrganismDetailsDialog.vue

```diff
@@ -0,0 +1,155 @@
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">Name</span>
+              <span class="mdc-list-item__secondary-text">{{ globalArticle.name }}</span>
+            </div>
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">Menge</span>
+              <span class="long-text">{{ article.amount }} {{ article.unit }}</span>
+            </div>
+          </div>
+        </div>
+        <hr>
+        <div class="align-container">
+          <div class="details-dialog--select-container">
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">Name</span>
+              <span class="mdc-list-item__secondary-text">{{ globalArticle.name }}</span>
+            </div>
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">EAN</span>
+              <span class="mdc-list-item__secondary-text">{{ globalArticle.ean }}</span>
+            </div>
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">Preis</span>
+              <span class="long-text">{{ article.price }} €</span>
+            </div>
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">Lagerort</span>
+              <span class="long-text">{{ storageLocation.name }}</span>
+            </div>
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">MHD + max. Überschreitung</span>
+              <span class="long-text">{{ (new Date(article.expirationDate * 1000)).toLocaleDateString() }} + {{ globalArticle.expiringMax }} Tage</span>
+            </div>
+            <div class="item-container">
+              <span class="mdc-list-item__primary-text">Produkt-Gruppe</span>
+              <span class="long-text">{{ productGroup.name }}</span>
+            </div>
+          </div>
+          <div>
+            <div>
+              <div class="naehrwert-tabelle">
+                <h2>Nährwerte</h2>
+                <table>
+                  <tr>
+                    <td class="long-text">Kalorien:</td>
+                    <td class="long-text">{{ globalArticle.calories }}</td>
+                  </tr>
+                  <tr>
+                    <td class="long-text">Kohlenhydrate:</td>
+                    <td class="long-text">{{ globalArticle.carbs }}</td>
+                  </tr>
+                  <tr>
+                    <td class="long-text">Eiweiß:</td>
+                    <td class="long-text">{{ globalArticle.protein }}</td>
+                  </tr>
+                  <tr>
+                    <td class="long-text">Fett:</td>
+                    <td class="long-text">{{ globalArticle.fat }}</td>
+                  </tr>
+                </table>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="item-container">
```

Comment:
Only display container if note is available
```suggestion
        <div 
          v-if="article.note"
          class="item-container"
        >
```